api/src/certification/configuration/application/complementary-certification-route.js

```diff
@@ -93,6 +96,40 @@ const register = async function (server) {
         ],
       },
     },
+    {
+      method: 'PATCH',
+      path: '/api/admin/complementary-certifications/{complementaryCertificationKey}/consolidated-framework',
+      config: {
+        pre: [
+          {
+            method: (request, h) =>
+              securityPreHandlers.hasAtLeastOneAccessOf([securityPreHandlers.checkAdminMemberHasRoleSuperAdmin])(
+                request,
+                h,
+              ),
+            assign: 'hasRoleSuperAdmin',
+          },
+        ],
+        validate: {
+          params: Joi.object({
+            complementaryCertificationKey: Joi.string().valid(...Object.values(ComplementaryCertificationKeys)),
+          }),
+          payload: Joi.object({
+            data: {
+              attributes: {
+                createdAt: Joi.date().required(),
+              },
+            },
+          }),
+        },
+        handler: complementaryCertificationController.calibrateConsolidatedFramework,
+        tags: ['api', 'admin'],
+        notes: [
+          'Cette route est restreinte aux utilisateurs authentifiés avec le rôle Super Admin',
+          'Elle permet de récupérer la calibration générée par Data pour la renseigner dans le référentiel cadre',
```

Comment:
La partie "Elle permet de récupérer" m'a fait penser d'abord a "tiens ca devrait etre un GET", mais c'est dans le usecase que l'on vient recup

```suggestion
          'Elle declenche la synchronization entre une calibration (contenu formatif) et son référentiel cadre (Pix)',
```